src/views/home/Index.svelte

```diff
@@ -70,14 +70,9 @@
         <div class="project">
           <Link
             route={{
-              resource: "projects",
-              params: {
-                view: { resource: "tree" },
-                id: project.id,
-                baseUrl,
-                peer: undefined,
-                revision: undefined,
-              },
+              resource: "project.tree",
+              project: project.id,
```

Comment:
Wonder if this property shouldn't be `projectId`, so it's clear what it is?

Reply:
I was thinking about that as well. In the end I went for `project` because of the following reasons:
* It’s typed as a string so you can’t assign a `Project` inadvertently.
* This property is part of a `Route` object where we always deal with identifiers.
* We use short forms for other identifiers as well: `commit` instead of `commitId`, `peer` instead of `peerId`, etc.